### PR TITLE
First draft of Release Notes for 5.5.2

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.5.md [hunk 1]
```diff
@@ -24,6 +24,52 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 ------
 - **Plugin Upgrades:** If you are running with plugins, updates are required for some plugins **before** upgrading the conductor to SSR version 5.4.0 or higher. Please review the [Plugin Configuration Generation Changes](intro_upgrade_considerations.md#plugin-configuration-generation-changes) for additional information.  
 
+## Release 5.5.2-1
+
+**Release Date:** June 30, 2022
+
+### New Features
+
+- **I95-40195 LDAP does not allow search base to be configured correctly:** This is a new feature. New commands, new config.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:33]: Is this a placeholder for documenting how one would go about configuring the feature?

File: docs/release_notes_128t_5.5.md [hunk 2]
```diff
@@ -24,6 +24,52 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+### New Features
+
+- **I95-40195 LDAP does not allow search base to be configured correctly:** This is a new feature. New commands, new config. 
+------
+- **I95-40333 Save credentials for accessing SSR software repositories:** `set software access-token` is a new PCLI command to save credentials for accessing SSR software repositories. This provides a way to run `install128t repo authenticate` without dropping to a linux shell.
+------
+- **I95-44863 Automatic Core Assignment after Reboot:** On systems where forwarding-core-mode is set to automatic, if the CPU core count changes the software will automatically recalculate the core count and allocation at reboot.
+------
+- **I95-46562 Allow targeting another router or node when saving
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:39]: looks like an unnecessary line break is here

File: docs/release_notes_128t_5.5.md [hunk 3]
```diff
@@ -24,6 +24,52 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+- **The following CVE's have been addressed and resolved:** I95-45060, I95-45123, I95-45165, I95-46020, I95-46359.
+------
+- **I95-45126 Split-brain after the sync interface goes down:** Resolved an issue that if the SSR software experienced a crash while it owned an interface from an X553 device, other devices hosted by the same chip could be impacted.
+------
+- **I95-45814 No Bandwidth statistics visible in GUI:** Resolved an issue when processing high numbers of services and service routes which prevented a subset of stats from being stored and displayed.
+------
+- **I95-46114 SSR flooded with Highway messages:** The chatty `InterfaceMap::Exception: Unable to find path to peer` highway log has been suppressed.
+------
+- **I95-46136 Unused app-id stats not being purged fast enough:** Ressolved an issue where app-id stats tracked per client, per app, per next-hop are not cleaned up when inactive.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:53]: Typo: Resolved

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.5.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.5.md:39]: fixed

File: docs/release_notes_128t_5.5.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.5.md:53]: fixed

File: docs/release_notes_128t_5.5.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.5.md]: this has been updated and contains the new information and link.

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**